src/sentry/api/endpoints/project_user_stats.py

```diff
@@ -5,20 +5,33 @@
 from rest_framework.response import Response
 
 from sentry.app import tsdb
+from sentry.api.base import EnvironmentMixin
 from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.models import Environment
 
 
-class ProjectUserStatsEndpoint(ProjectEndpoint):
+class ProjectUserStatsEndpoint(EnvironmentMixin, ProjectEndpoint):
     def get(self, request, project):
+        try:
+            environment_id = self._get_environment_id_from_request(
+                request,
+                project.organization_id,
+            )
+        except Environment.DoesNotExist:
+            raise ResourceDoesNotExist
+
         now = timezone.now()
         then = now - timedelta(days=30)
 
+        # TODO(tkaemming): Rollup doesn't actually work correctly here
```

Comment:
This is a distinct counter so summing values can result in double counting unique values

Reply:
Assume this will be a different PR?

Reply:
Yeah, that was my plan (though admittedly I probably will forget)